src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { memo, useState } from 'react';
+import cN from 'classnames';
+import { Todo } from '../../types/Todo';
+import { TodoPatch } from '../../types/TodoPatch';
+
+type Props = {
+  todo: Todo,
+  onDelete: (todoId: number) => void,
+  onUpdateTodo?: (todoId: number, data: TodoPatch) => void,
+  isLoading?: boolean
+};
+
+export const TodoItem: React.FC<Props> = memo(
+  ({
+    todo,
+    onDelete,
+    onUpdateTodo = () => { },
+    isLoading = true,
```

Comment:
Default value for `isLoading` prop is set to `true`. It is better to set default to `false`, as usually you don't want to show loading state until a request hasn't started.